static/js/map.js

```diff
@@ -0,0 +1,198 @@
+import layers from '../style/layers.js'
+import { layersArray, layersDefinitions, layersVisibilityState } from './layers.js';
+import { controlsLocation, modesSettings, sidebarDivId } from './constants.js';
+import { geocoder_api } from './geocoder_api.js';
+import { disableLayer, enableLayer, renderBasicPopup } from './functions.js';
+import { legendFactory } from './legend.js';
+import { bulmaBootstrap } from './bulma_init.js';
+
+legendFactory(layersArray, layersVisibilityState);
+document.addEventListener('DOMContentLoaded', function() { bulmaBootstrap(getMode); });
+
+function toggleMode(map) {
+  const modes = {
+    'forUkrainian': 'forPole',
+    'forPole': 'forUkrainian',
+  }
+  const newMode = modes[getMode()];
+  const settings = modesSettings[newMode];
+  settings.layersToDisable.forEach(layerGroupId => disableLayer(map, layerGroupId));
+  settings.layersToEnable.forEach(layerGroupId => enableLayer(map, layerGroupId));
+  saveMode(newMode);
+}
+
+function getMode() {
+  let mode;
+  if (typeof localStorage !== 'undefined') {
+    mode = localStorage.getItem('mode');
+  }
+
+  if (!mode) {
+    return 'forUkrainian';
+  }
+
+  return mode;
+}
+
+function saveMode(mode) {
+  localStorage.setItem('mode', mode);
+}
+
+function loadCenter() {
+  let center;
+  if (typeof localStorage !== 'undefined') {
+    center = localStorage.getItem("center");
+    if (center) try {
+      center = JSON.parse(center);
+    } catch (e) { console.warn(e) }
+  }
+
+  if (!center || !center.lng || typeof center.lng !== 'number' || !center.lat || typeof center.lat !== 'number')
```

Comment:
we are using in multiple places the same check for number -> extract it to the  extarnal helper function maybe?

Reply:
Actually in this particular place I thought it would be better to just initialize the `center` variable with some default value and update only if there is some other value in the `localStorage`.